torch/csrc/jit/serialization/pickler.h

```diff
@@ -192,7 +193,6 @@ class TORCH_API Pickler {
       const std::string& class_name);
   // raw string data is appended directly to the byte stream
   void pushBytes(const std::string& string);
-  void pushTensorData(const at::Tensor& tensor);
```

Comment:
Why deleting this line?

Reply:
It is not used in the PyTorch repo / its deadcode

Reply:
Shall we move this change to a separate PR? We usually would avoid sneaking in irrelevant changes in PRs.

Reply:
Okay, I'll make a pr for this.